mapbox/services/geocoding.py

```diff
@@ -84,6 +94,11 @@ def geojson():
         return resp
 
     @property
+    def country_codes(self):
+        """A list of valid country codes"""
+        return ('ad', 'ae', 'al', 'ao', 'ar', 'at', 'au', 'ba', 'be', 'bg', 'bh', 'bn', 'br', 'bw', 'by', 'ca', 'ch', 'cl', 'cn', 'co', 'cu', 'cy', 'cz', 'de', 'dk', 'dz', 'ee', 'eg', 'es', 'fi', 'fr', 'gb', 'gf', 'gp', 'gr', 'hr', 'hu', 'id', 'ie', 'in', 'is', 'it', 'jo', 'jp', 'ke', 'kw', 'lb', 'lt', 'lu', 'lv', 'ma', 'md', 'me', 'mk', 'mt', 'mx', 'my', 'nl', 'no', 'nz', 'pe', 'ph', 'pl', 'pt', 'ro', 'rs', 'ru', 'se', 'sg', 'si', 'sk', 'sm', 'th', 'tr', 'tw', 'ua', 'us', 'uy', 've', 'vn', 'za')
```

Comment:
We might want to consider other ways for defining the country codes
* A separate text file that is read in by `country_codes`
* A separate module or use the existing [iso3166 module](https://pypi.python.org/pypi/iso3166)

At the very least we should reformat this tuple to be < 80 chars wide so we don't get wrapping.